docker/datahub-gms/debug/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM openjdk:8-jre-alpine
```

Comment:
Add a comment that this is a dummy docker file for quick debugging purposes

Reply:
Done